frontend/tailwind.config.js

```diff
@@ -1,36 +1,27 @@
+const defaultTheme = require('tailwindcss/defaultTheme')
+
 module.exports = {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
-    //   theme: {
-    //     screens: {
-    //       sm: '480px',
-    //       md: '768px',
-    //       lg: '976px',
-    //       xl: '1440px',
-    //     },
-    //     colors: {
-    //       'blue': '#1fb6ff',
-    //       'purple': '#7e5bef',
-    //       'pink': '#ff49db',
-    //       'orange': '#ff7849',
-    //       'green': '#13ce66',
-    //       'yellow': '#ffc82c',
-    //       'gray-dark': '#273444',
-    //       'gray': '#8492a6',
-    //       'gray-light': '#d3dce6',
-    //     },
-    //     fontFamily: {
-    //       sans: ['Graphik', 'sans-serif'],
-    //       serif: ['Merriweather', 'serif'],
-    //     },
-    //     extend: {
-    //       spacing: {
-    //         '128': '32rem',
-    //         '144': '36rem',
-    //       },
-    //       borderRadius: {
-    //         '4xl': '2rem',
-    //       }
-    //     }
-    //   },
+    theme: {
+        extend: {
+            colors: {
+                orange: {
+                    default: '#fc9143',
+                },
+                gray: {
+                    dark: '#303030',
+                    default: '#606060',
+                },
+                blue: {
+                    default: '#0097ec',
+                },
+                'primary-orange': '#fc9143',
+            },
+            fontFamily: {
+                sans: ['PT Sans', ...defaultTheme.fontFamily.sans],
+                mplus: ['"M PLUS Rounded 1c"', ...defaultTheme.fontFamily.sans],
```

Comment:
I'm following how you set up tailwind.config. So `webfontloader` loads the m plus font itself and the `fontFamily.mplus` property will create/reference the css utility class `font-mplus`?

Reply:
Yup that's correct!